src/Symfony/Component/Console/Output/OutputInterface.php

```diff
@@ -33,17 +33,17 @@ interface OutputInterface
     /**
      * Writes a message to the output.
      *
-     * @param bool                             $newline Whether to add a newline
-     * @param self::VERBOSITY_*|self::OUTPUT_* $options A bitmask of options (one of the OUTPUT or VERBOSITY constants),
-     *                                                  0 is considered the same as self::OUTPUT_NORMAL | self::VERBOSITY_NORMAL
+     * @param bool                                          $newline Whether to add a newline
+     * @param int-mask-of<self::VERBOSITY_*,self::OUTPUT_*> $options A bitmask of options (one of the OUTPUT or VERBOSITY constants),
```

Comment:
Not to my knowledge nor experience actually ; the `int` typehint is still in use for the quick documentation, autocomplete or inspections, but I don't know if it can cause quirks with some plugins.

Reply:
the EAP version of PHPStorm has full support for parsing constant wildcards (the stable release supports the case with a name prefix but not the full wildcard on the class IIRC). But indeed, it will fallback to `int` in such case.